input/pagecontent/general-requirements.md

```diff
@@ -89,13 +89,13 @@ How the system processes the resource depends on local requirements that could a
 
 When a *Must Support* element requires a more tightly stated obligation, this obligation is specified in the AU Core Requester [obligation extension](https://hl7.org/fhir/extensions/StructureDefinition-obligation.html) on the element definition.
 
-#### Presentation of elements labelled Must Support in profiles
+#### Presentation of Must Support and obligations in profiles
 
-##### Presentation of elements labelled Must Support in table views
+##### Presentation of Must Support and obligations in table views
```

Comment:
Capitalise Obligation. Headings should be 
- Presentation of elements labelled Must Support and Obligation in profiles
- Presentation of elements labelled Must Support and Obligation in table views